### PR TITLE
PP-10004 M1 support for integration tests

### PR DESCRIPTION
**Pull request by nlsteers**
## WHAT YOU DID

- removed use of `govuk/postgres:11`, baselining on architecture agnostic `postgres:11.16`
- updated test bases and rules to enable cross architecture support
- added integration tests specific liquibase changelog (`it-migrations.xml`) to initialise the database before migrations can be run
- updated local sqs image to be consistent with other Pay applications
- updated GHA to use new images

### CODE REVIEW

**Comment by oswaldquek** [comment]
What's the purpose of it-migrations.xml?

File: src/test/java/uk/gov/pay/adminusers/infra/SqsTestDocker.java [hunk 1]
```diff
@@ -31,19 +31,13 @@ private static void createContainer() {
         if (sqsContainer == null) {
             logger.info("Creating SQS Container");
 
-            sqsContainer = new GenericContainer("mvisonneau/alpine-sqs:1.2.0")
+            sqsContainer = new GenericContainer("softwaremill/elasticmq-native")
```
Comment (oswaldquek) [inline src/test/java/uk/gov/pay/adminusers/infra/SqsTestDocker.java:34]: Is this container change related to M1 support? If it is, it'd be good to write an explanation in the commit message.

File: .github/workflows/_run-pact-provider-tests.yml [hunk 2]
```diff
@@ -47,7 +47,7 @@ jobs:
           restore-keys: ${{ runner.os }}-m2
       - name: Pull docker image dependencies
         run: |
-          docker pull govukpay/postgres:11.1
+          docker pull postgres:11.16
```
Comment (oswaldquek) [inline .github/workflows/_run-pact-provider-tests.yml:50]: Is this container change related to M1 support? If it is, it'd be good to write an explanation in the commit message.

File: src/test/java/uk/gov/pay/adminusers/infra/SqsTestDocker.java (hunk 1 above)
Comment (nlsteers) [reply src/test/java/uk/gov/pay/adminusers/infra/SqsTestDocker.java:34]: hopefully the commit message if more clarifying now.

**Review by oswaldquek — APPROVED** [review]
@nlsteers I don't have an M1 sufficiently set up so can't verify the PR improves running the integration tests but assuming you have tested it the PR looks good.

**Comment by katstevens** [comment]
(Intel) `mvn clean verify` 

On branch: 5 min 40. 
On master: 5 min 52!

**Review by katstevens — APPROVED** [review]
